logs/logs.go

```diff
@@ -31,8 +31,6 @@ func GetLogs(rw http.ResponseWriter, req *http.Request) error {
 
 	if found {
 		tail = strconv.Itoa(int(tailTemp))
```

Comment:
Well i thought the 100 was there lol. Didnt realize i deleted it. I had it in 2 places. You are right i derped. Been a rough day